client/FxAccountClient.js

```diff
@@ -1207,14 +1207,23 @@ define([
    *
    * @method smsStatus
    * @param {String} sessionToken SessionToken obtained from signIn
+   * @param {Object} [options={}] Options
+   *   @param {String} [options.country] country Country to force. Will be used to test
```

Comment:
nit: can the comment for this option be cleaned up a bit? `Country to force for testing`, `Country to force and check if it is supported` ?